akka-actor/src/main/scala/akka/dispatch/Mailboxes.scala

```diff
@@ -214,9 +203,6 @@ private[akka] class Mailboxes(
       case null =>
         // It doesn't matter if we create a mailbox type configurator that isn't used due to concurrent lookup.
         val newConfigurator = id match {
-          // TODO RK remove these two for Akka 2.3
-          case "unbounded"                               => UnboundedMailbox()
-          case "bounded"                                 => new BoundedMailbox(settings, config(id))
```

Comment:
This is also documented in mailboxes.md. Even though seems to be a very old leftover I'd rather not change this in a patch release. You never know...

Reply:
@patriknw Thank you for your explanation. I can close this PR. It is not a problem. Should I do it or should it be opened for some time?

Reply:
Thanks, we can look at this again when it's time for 2.7. I'll close it for now.